packages/clerk-js/src/ui/components/KeylessPrompt/index.tsx

```diff
@@ -153,14 +159,15 @@ const _KeylessPrompt = (_props: KeylessPromptProps) => {
 
             <p
               data-text='Clerk is in keyless mode'
```

Comment:
I think you also need to change this property like `aria-label`?

Reply:
You mean the `data-text` ? That one's used for the yellow text animation

Reply:
Yeah. And once you change the text itself, the text and `data-text` are out of sync and the `::before` and `::after` are incorrect?